buzzmobile/tests/sense/test_inputer.py

```diff
@@ -0,0 +1,55 @@
+import rostest
+import unittest
+import rospy
+import time
+from std_msgs.msg import String
+import roslaunch
+
+PKG = 'buzzmobile'
+NAME = 'test_inputer'
```

Comment:
I thought that ros could infer package names from directories (which is how it actually infers `buzzmobile` for everything else, I'm pretty sure) but if there's no way to get this then ok.